modules/ROOT/pages/event-tracking.adoc

```diff
@@ -151,26 +162,23 @@ services:
       trackingLevel: TRACKING
       replayLevel: NONE
 ----
-In addition to changing the `trackingLevel` and `replayLevel` for a Mule application, you can also change these settings for flows inside the listed Mule application by adding the `trackedFlows` value.
 
-The `trackedFlows` element contains additional elements to specify granular tuning of what should be tracked for a flow, and how any message replay should behave. When a flow is configured, these settings override the settings inherited by the global and Mule application level settings.
+==== Configure Tracking for Flows in a Mule App
 
-==== Configuring Tracking per Flow of a Mule Application
+You can override global and app-level settings for flows in the specified Mule app by adding the `trackedFlows` element. This element specifies what to track for a flow, and how message replay behaves.
 
-In addition to the Mule application level `trackingLevel` and `replayLevel` settings, you can also list one or more flows and override/customize the global and application level settings for each flow.
-
-The tracked flows options are described in this table:
+This table describes the tracked flows options:
 
 [%header%autowidth.spread]
 |===
 |Tracked Flow Option | Description
-|`flow name` |  The name of the flow, as it is coded in the Mule application's XML configuration file.
-|`tracking level` |  One of the values `NONE`, `BE`, `TRACKING`, or `DEBUG`, as described in the previous table.
-|`replayLevel` |  One of the values `NONE`, `SOURCED` (only store messages if the flow begins with a message source), `ALL`
+|`flow name` |  The name of the flow, as it is coded in the Mule app XML configuration file.
+|`tracking level` |  One of the values `NONE`, `BE`, `TRACKING`, or `DEBUG`. See <<Event Tracking Levels>>.
+|`replayLevel` |  One of the values `NONE`, `SOURCED`, or `ALL`. See the <<Replay,Replay Levels>> table.
 |`payloadExcluded` |  If `replayLevel` is `SOURCED` or `ALL`, this determines whether to also store the message payload for later replay, or to just store the message metadata.
 |===
 
-Here is an example that adds some flow level tracking configuration to the previous `mule-flights` Mule application configuration. The global and Mule application level tracking is using the default values (`NONE`).
+This example adds flow-level tracking configuration to the `mule-flights` Mule app configuration. The global and Mule app-level tracking use the default values (`NONE`).
```

Comment:
"This example adds flow-level tracking to 'mule-flights.'"

"...use the default values (`NONE`)."    Maybe I am lost, but...was the default value set to NONE in a previous example, or is it true that the actual default value for all tracking is NONE?